client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -201,8 +199,10 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		case 'not-authorized':
 			return (
 				<NotAuthorized
-					onStartBuilding={ onNotAuthorizedClick }
-					onStartBuildingText={ translate( 'Skip to dashboard' ) }
+					type="source-site-not-connected"
+					sourceSiteUrl={ sourceSiteUrl }
+					targetSiteUrl={ targetSite.URL }
+					startImport={ startImport }
```

Comment:
This seems to be the wrong function here, when I click on the CTA, it doesn't do anything.

Reply:
As we agreed, the CTA will be a page refresh.